### PR TITLE
add schema/docs for `editor.suggest.selectQuickSuggestions` setting

### PR DESCRIPTION
**Pull request by jrieken**
https://github.com/microsoft/vscode/issues/139825

### CODE REVIEW

**Review by lramos15 — APPROVED** [review]

File: src/vs/editor/common/config/editorOptions.ts [hunk 1]
```diff
@@ -4244,6 +4244,11 @@ class EditorSuggest extends BaseEditorOption<EditorOption.suggest, ISuggestOptio
 					default: defaults.shareSuggestSelections,
 					markdownDescription: nls.localize('suggest.shareSuggestSelections', "Controls whether remembered suggestion selections are shared between multiple workspaces and windows (needs `#editor.suggestSelection#`).")
 				},
+				'editor.suggest.selectQuickSuggestions': {
+					type: 'boolean',
+					default: defaults.selectQuickSuggestions,
+					markdownDescription: nls.localize('suggest.selectQuickSuggestions', "Controls whether the suggest becomes active when triggered via quick suggest or trigger characters. ")
```
Comment (Tyriar) [inline src/vs/editor/common/config/editorOptions.ts:4250]: There's an extra space at the end

**Review by Tyriar — DISMISSED** [review]

File: src/vs/editor/common/config/editorOptions.ts (hunk 1 above)
Comment (lramos15) [inline src/vs/editor/common/config/editorOptions.ts:4250]: Should this say "whether the suggestion becomes.."

**Review by lramos15 — DISMISSED** [review]

File: src/vs/editor/common/config/editorOptions.ts (hunk 1 above)
Comment (jrieken) [reply src/vs/editor/common/config/editorOptions.ts:4250]: thanks

Comment (jrieken) [reply src/vs/editor/common/config/editorOptions.ts:4250]: thanks

**Review by lramos15 — APPROVED** [review]

**Pull request merged**